environments/aks/prod.tfvars

```diff
@@ -1,7 +1,7 @@
 cluster_count                      = 2
 kubernetes_cluster_version         = "1.23"
-kubernetes_cluster_agent_min_count = "70"
-kubernetes_cluster_agent_max_count = "120"
+kubernetes_cluster_agent_min_count = "50"
```

Comment:
I think these are not used, can u pls remove them across to avoid confusion

Reply:
Sure thing